<h1 id="introduction">Introduction</h1>
<p>Since the discovery that the world in which we live is entirely filled with organisms too small to see with the eye, biologists have drawn a line between the microorganism and the multicellular organism. However, the search for the evolutionary apex of cellular organisms has blurred this line, from the theory of lateral gene flow <span class="citation" data-cites="Woe98">(Woese 1998)</span> to the discovery that archaea and eukarya share more in common with each other than either shares with bacteria <span class="citation" data-cites="Woe77">(Woese and Fox 1977)</span>. In researching the microbiome – the physical commensal relationship between animals and their microbial inhabitants – it is important to consider the full diversity of microbial taxa. This paper seeks to investigate biases in microbiome research toward specific microbial taxa and seek out how the less-studied varieties of microbial inhabitants may have effects unaccounted for.</p>
<p>Despite a growth of research interest in archaea since its identification by <span class="citation" data-cites="Woe77">Woese and Fox (1977)</span>, little research has been conducted into their relationships to other organisms, such as humans. As the domain was originally understood as a clade of extremophiles due to their original isolation from geothermal vents <span class="citation" data-cites="DeL01">(DeLong and Pace 2001)</span>, it would be expected that archaea would be a minor part of more familiar environments; however, it is now known that members of archaea exist with great frequency in non-extreme environments, such as plankton, lakes, and soils <span class="citation" data-cites="DeL01">(DeLong and Pace 2001)</span>. Regardless, research into the composition of human microbiota retains a strong bias toward bacteria <span class="citation" data-cites="Llo16 The12">(Lloyd-Price, Abu-Ali, and Huttenhower 2016; Huttenhower et al. 2012)</span>. Comparative primatology has often failed to expand microbiota research beyond bacteria, frequently excluding unknown or infrequent phyla from results of DNA extraction <span class="citation" data-cites="Gar19 Mal18 Ork191 Ork19">(Garber et al. 2019; Mallott and Amato 2018; Orkin et al. 2019; Orkin, Webb, and Melin 2019)</span>. At its extreme, the term “microbe” has been used synonymously with the term “bacteria” despite the far more precise definition of the latter <span class="citation" data-cites="Mal18">(for an example, see Mallott and Amato 2018)</span>.</p>
<h1 id="analysis-of-previous-research">Analysis of previous research</h1>
<p>In order to assay the relative biases of microbiome research it would be necessary to compare the references to microbial phyla in published literature to the prevalence of such microbes in microbiomes. As much research into the human microbiome is undertaken from a medical perspective, NCBI PubMed contains a wealth of full-text and abstracts in its catalog, which is easily indexed for specific terms and phrases. Thus, NCBI PubMed’s article archive was chosen as a sample for language analysis.</p>
<h2 id="methods">Methods</h2>
<p>A search for the exact phrase “human microbiome” was conducted on NCBI PubMed in October 2019, fetching 1,779 results, which were downloaded as an XML file. Using R version 3.6.1 <span class="citation" data-cites="RCo19">(R Core Team 2019)</span> on Microsoft Windows 10 18362.418, these results were indexed with <code>pubmed.mineR</code> version 1.0.16 <span class="citation" data-cites="Ran19">(Rani, Ramachandran, and Shah 2019)</span> and queried for metadata from the NCBI PubTator service <span class="citation" data-cites="Wei19">(Wei et al. 2019)</span>. PubTator analyzes the text of PubMed abstracts and full-text articles to determine the names of species and taxonomic tanks mentioned in such articles. Excluding mentions to <em>Homo sapiens</em>, the taxonomic ranks were looked up in the NCBI Taxonomy Database <span class="citation" data-cites="Nat19">(<em>NCBI Taxonomy Database</em> 2019)</span> using <code>ncbitax2lin</code> <span class="citation" data-cites="Xue19">(Xue 2019)</span> to convert the database to CSV.</p>
<p>References to taxonomic ranks more specific than phylum were simplified to phyla, and less specific references were recorded as “<em>domain</em> (nonspecific).” In the case of viruses, which have separate ranking system, the NCBI value “no rank – 0” was used in place of phylum. The number of references to each phylum was totaled and categorized by domain. All R code executed is presented in .</p>
<h2 id="results">Results</h2>
<p>Of the 1,779 articles indexed, 1,730 mentioned a taxonomic rank or species. This totaled 5,354 individual mentions, of which 2,419 were to species or ranks other than humans. As visible in , bacteria made up the majority of references, with the phyla Proteobacteria and Firmicutes second only to nonspecific references to bacteria as a domain. Eukarya were second to bacteria in total number of references; however, the most frequently discussed phylum within the domain was Chordata, which likely stems from discussion of vertebrate microbiomes such as Mus musculus rather than indicating the presence of chordates in the human microbiome. Viruses were referenced moderately frequently, with nonspecific virus references making up the greatest proportion of the domain’s coverage, followed by bacteriophages. Finally, archaea were referenced least frequently of all, with nine total references amounting to less than any viral phylum.</p>
<h1 id="the-role-of-non-bacterial-organisms-in-the-microbiome">The Role of Non-Bacterial Organisms in the Microbiome</h1>
<h2 id="eukarya">Eukarya</h2>
<p>Yeasts have been found to colonize infant guts gradually, with an isolation rate of 13% at 28-46 days old <span class="citation" data-cites="Ben84">(Benno, Sawada, and Mitsuoka 1984)</span> and 50% as soon as four months of age <span class="citation" data-cites="Ell75">(Ellis-Pegler, Crabtree, and Lambert 1975)</span>. Notably, this rate of colonization was consistent between breast-fed and formula-fed infants, in contrast to the colonization rates of bacterial phyla which differ significantly between the two feeding mechanisms. From a life-history perspective, this presents a novel question of how yeast colonize the infant gut and whether the mechanism by which breastmilk contains microbiota associated with the mother’s gut microbiome <span class="citation" data-cites="Mar13">(Martin and Sela 2013)</span> is also capable of providing yeast, whose eukaryotic cells differ in size by orders of magnitude from prokaryotic cells.</p>
<p>However, research conducted on yeast inhabitants of animals focuses primarily on their pathological role. The genus <em>Candida</em> is well known for its role in infectious disease in humans, particularly in the immunocompromised <span class="citation" data-cites="Kou11">(Kourkoumpetis et al. 2011)</span>. However, species of yeast, including many <em>Candida</em> spp., have frequently been identified on the human body, including the vagina, gut, and skin, without any associated mycosis <span class="citation" data-cites="Man10">(Manolakaki et al. 2010)</span>. Even for species known to be infectious agents, between 12% <span class="citation" data-cites="Cho86">(Chow et al. 1986)</span> and 80% <span class="citation" data-cites="Soe07">(Soebel 2007)</span> of asymptomatic healthy women are vaginal carriers. In place of viewing yeast through the lens of infection, it should be considered as a part of the ecosystem that makes up the human body. How it reaches and colonizes infants, and why its prevalence makes up far less of the population than do bacterial commensals, merits further study.</p>
<h2 id="archaea">Archaea</h2>
<p>In 1966 it was discovered that humans both respire and flatulate methane, which is not known to be biologically produced by any bacteria or eukaryote. Following this observation, methanogenic prokaryotes, from what would later be called the archaea, were identified in human feces <span class="citation" data-cites="Not68">(Nottingham and Hungate 1968)</span>. The archaeal domain remains the only taxon with members known to be biological producers of methane <span class="citation" data-cites="Tha06">(Thauer and Shima 2006)</span>, and as such is unique in its residence within the human gut. Despite widespread colonization by a diverse array of archaeal species, the ecological niche which they occupy within the human body and whether they are transferred vertically or environmentally remains unknown <span class="citation" data-cites="Dri11">(Dridi, Raoult, and Drancourt 2011)</span>. It has been theorized that many more lineages of archaea remain undiscovered, as the DNA analysis methods used to identify bacterial genomes are incompatible with archaeal cell walls <span class="citation" data-cites="Hor15">(Horz 2015)</span>.</p>
<h2 id="viruses">Viruses</h2>
<p>Despite not fitting many definitions of biological life, viruses are the most diverse form of biological material on the planet. They are known to outnumber bacteria by a factor of nearly ten <span class="citation" data-cites="Ore97">(Oren, Bratbak, and Heldal 1997)</span> and are a major driver of evolution in prokaryotes. Although humans are susceptible to a wide variety of viruses, the viral community in the microbiome, so fast it has itself been called a “virome” <span class="citation" data-cites="Wyl12">(Wylie, Weinstock, and Storch 2012)</span>, is dominated primarily by bacteriophages whose sequences are novel <span class="citation" data-cites="Dut14">(Dutilh et al. 2014)</span>. The phages identifiable by genetic sequence are primarily those known to attack bacterial phyla associated with the microbiome, yet appear to be vastly more diverse than the virome of the ocean <span class="citation" data-cites="Wal14">(Waller et al. 2014)</span>. Unlike the theories of bacterial colonization, how such a vast and dynamic virome is able to establish itself by adulthood remains unknown. The effect of age, and whether milk carries a virome of similar diversity would be intriguing avenues for further study.</p>
<h1 id="limitations">Limitations</h1>
<p>Utilizing both life-history and ecological frameworks for approaching non-bacterial components of the microbiome, it is clear that the roles such taxa play within our bodies is underexplored. While this paper sought to touch on these issues, it is limited by both methodology and its brevity. The use of NCBI PubMed for language analysis suffers from a bias toward medical research, and so microbiome research will often consider the roles of known pathological agents rather than in-depth discussions of all possible inhabitants. Eukarya is also a domain of organisms highly diverse in physical makeup and metabolic strategy, while only yeasts were discussed here. The roles of other unicellular parasites (for instance, <em>Toxoplasma gondii</em>) and of helminthic worms, which each form diverse monophyletic groups, can also be explored via the lens of the microbiome’s life-history. More research is needed in this area to investigate how the diverse community of microbes within humans evolved and what role it plays even in the absence of pathology.</p>
<p><span class="citation" data-cites="Ran19 Wic18 Xue19">(Rani, Ramachandran, and Shah 2019; Wickham, Hester, and Francois 2018; Xue 2019)</span></p>
<div id="refs" class="references hanging-indent" role="doc-bibliography">
<div id="ref-Ben84">
<p>Benno, Yoshimi, Ken Sawada, and Tomotari Mitsuoka. 1984. “The Intestinal Microflora of Infants: Composition of the Fecal Flora in Breast-Fed and Bottle-Fed Infants.” <em>Microbiology and Immunology</em> 28 (9): 975–86. <a href="https://doi.org/10.1111/j.1348-0421.1984.tb00754.x">https://doi.org/10.1111/j.1348-0421.1984.tb00754.x</a>.</p>
</div>
<div id="ref-Cho86">
<p>Chow, Anthony W., Robin Percival-Smith, Karen H. Bartlett, Anita M. Goldring, and Brenda J. Morrison. 1986. “Vaginal Colonization with Escherichia Coli in Healthy Women: Determination of Relative Risks by Quantitative Culture and Multivariate Statistical Analysis.” <em>American Journal of Obstetrics and Gynecology</em> 154 (1): 120–26. <a href="https://doi.org/10.1016/0002-9378(86)90406-0">https://doi.org/10.1016/0002-9378(86)90406-0</a>.</p>
</div>
<div id="ref-DeL01">
<p>DeLong, Edward F., and Norman R. Pace. 2001. “Environmental Diversity of Bacteria and Archaea.” <em>Systematic Biology</em> 50 (4): 470–78. <a href="https://doi.org/10.1080/10635150118513">https://doi.org/10.1080/10635150118513</a>.</p>
</div>
<div id="ref-Dri11">
<p>Dridi, Bédis, Didier Raoult, and Michel Drancourt. 2011. “Archaea as Emerging Organisms in Complex Human Microbiomes.” <em>Anaerobe</em> 17 (2): 56–63. <a href="https://doi.org/10.1016/j.anaerobe.2011.03.001">https://doi.org/10.1016/j.anaerobe.2011.03.001</a>.</p>
</div>
<div id="ref-Dut14">
<p>Dutilh, Bas E., Noriko Cassman, Katelyn McNair, Savannah E. Sanchez, Genivaldo G. Z. Silva, Lance Boling, Jeremy J. Barr, et al. 2014. “A Highly Abundant Bacteriophage Discovered in the Unknown Sequences of Human Faecal Metagenomes.” <em>Nature Communications</em> 5: 4498. <a href="https://doi.org/10.1038/ncomms5498">https://doi.org/10.1038/ncomms5498</a>.</p>
</div>
<div id="ref-Ell75">
<p>Ellis-Pegler, R. B., C. Crabtree, and H. P. Lambert. 1975. “The Faecal Flora of Children in the United Kingdom.” <em>Epidemiology &amp; Infection</em> 75 (1): 135–42. <a href="https://doi.org/10.1017/S002217240004715X">https://doi.org/10.1017/S002217240004715X</a>.</p>
</div>
<div id="ref-Gar19">
<p>Garber, Paul Alan, Elizabeth K. Mallott, Leila M. Porter, and Andres Gomez. 2019. “The Gut Microbiome and Metabolome of Saddleback Tamarins (<em>Leontocebus Weddelli</em>): Insights into the Foraging Ecology of a Small‐bodied Primate.” <em>American Journal of Primatology</em> Upcoming: e23003. <a href="https://doi.org/10.1002/ajp.23003">https://doi.org/10.1002/ajp.23003</a>.</p>
</div>
<div id="ref-Hor15">
<p>Horz, Hans-Peter. 2015. “Archaeal Lineages Within the Human Microbiome: Absent, Rare, or Elusive?” <em>Life</em> 5 (2): 1333–45. <a href="https://doi.org/10.3390/life5021333">https://doi.org/10.3390/life5021333</a>.</p>
</div>
<div id="ref-The12">
<p>Huttenhower, Curtis, Dirk Gevers, Rob Knight, Sahar Abubucker, Jonathan H. Badger, Asif T. Chinwalla, Heather H. Creasy, et al. 2012. “Structure, Function and Diversity of the Healthy Human Microbiome.” <em>Nature</em> 486: 207–14. <a href="https://doi.org/10.1038/nature11234">https://doi.org/10.1038/nature11234</a>.</p>
</div>
<div id="ref-Kou11">
<p>Kourkoumpetis, Themistoklis K., George C. Velmahos, Panayiotis D. Ziakas, Emmanouil Tampakakis, Dimitra Manolakaki, Jeffrey J. Coleman, and Eleftherios Mylonakis. 2011. “The Effect of Cumulative Length of Hospital Stay on the Antifungal Resistance of Candida Strains Isolated from Critically Ill Surgical Patients.” <em>Mycopathologica</em> 171 (2): 85–91. <a href="https://doi.org/10.1007/s11046-010-9369-3">https://doi.org/10.1007/s11046-010-9369-3</a>.</p>
</div>
<div id="ref-Llo16">
<p>Lloyd-Price, Jason, Galeb Abu-Ali, and Curtis Huttenhower. 2016. “The Healthy Human Microbiome.” <em>Genome Medicine</em> 8: 51. <a href="https://doi.org/10.1186/s13073-016-0307-y">https://doi.org/10.1186/s13073-016-0307-y</a>.</p>
</div>
<div id="ref-Mal18">
<p>Mallott, Elizabeth K., and Katherine R. Amato. 2018. “The Microbial Reproductive Ecology of White-Faced Capuchins (<em>Cebus Capucinus</em>).” <em>American Journal of Primatology</em> 80 (8): e22896. <a href="https://doi.org/10.1002/ajp.22896">https://doi.org/10.1002/ajp.22896</a>.</p>
</div>
<div id="ref-Man10">
<p>Manolakaki, Dimitra, George C. Velmahos, Themistoklis Kourkoumpetis, Yuchiao Chang, Hasan B. Alam, Marc M. De Moya, and Eleftherios Mylonakis. 2010. “Candida Infection and Colonization Among Trauma Patients.” <em>Virulence</em> 1 (5): 367–75. <a href="https://doi.org/10.4161/viru.1.5.12796">https://doi.org/10.4161/viru.1.5.12796</a>.</p>
</div>
<div id="ref-Mar13">
<p>Martin, Melanie A., and David A. Sela. 2013. “Infant Gut Microbiota: Developmental Influences and Health Outcomes.” In <em>Building Babies: Primate Development in Proximate and Ultimate Perspective</em>, edited by Kathryn B. H. Clancy, Katie Hinde, and Julienne N. Rutherford, 233–56. New York: Springer. <a href="https://doi.org/10.1007/978-1-4614-4060-4_11">https://doi.org/10.1007/978-1-4614-4060-4_11</a>.</p>
</div>
<div id="ref-Nat19">
<p><em>NCBI Taxonomy Database</em>. 2019. National Institutes of Health. <a href="https://www.ncbi.nlm.nih.gov/taxonomy">https://www.ncbi.nlm.nih.gov/taxonomy</a>.</p>
</div>
<div id="ref-Not68">
<p>Nottingham, P. M., and R. E. Hungate. 1968. “Isolation of Methanogenic Bacteria from Feces of Man.” <em>Journal of Bacteriology</em> 96 (6): 2178–9.</p>
</div>
<div id="ref-Ore97">
<p>Oren, Aharon, Gunnar Bratbak, and Mikal Heldal. 1997. “Occurrence of Virus-Like Particles in the Dead Sea.” <em>Extremophiles</em> 1 (3): 143–49. <a href="https://doi.org/10.1007/s007920050027">https://doi.org/10.1007/s007920050027</a>.</p>
</div>
<div id="ref-Ork191">
<p>Orkin, Joseph Daniel, Fernando A. Campos, Monica S. Myers, Saul E. Cheves Hernandez, Adrián Guadamuz, and Amanda D. Melin. 2019. “Seasonality of the Gut Microbiota of Free-Ranging White-Faced Capuchins in a Tropical Dry Forest.” <em>The ISME Journal</em> 13 (1): 183–96.</p>
</div>
<div id="ref-Ork19">
<p>Orkin, Joseph Daniel, Shasta Ellen Webb, and Amanda Dawn Melin. 2019. “Small to Modest Impact of Social Group on the Gut Microbiome of Wild Costa Rican Capuchins in a Seasonal Forest.” <em>American Journal of Primatology</em> Upcoming: e22985. <a href="https://doi.org/10.1002/ajp.22985">https://doi.org/10.1002/ajp.22985</a>.</p>
</div>
<div id="ref-Ran19">
<p>Rani, Jyoti, S. Ramachandran, and Ab. Rauf Shah. 2019. <em><code>pubmed.mineR</code>: Text Mining of Pubmed Abstracts</em>. Package version 1.0.16. <a href="https://cran.R-project.org/web/packages/pubmed.mineR/">https://cran.R-project.org/web/packages/pubmed.mineR/</a>.</p>
</div>
<div id="ref-RCo19">
<p>R Core Team. 2019. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://R-project.org/">https://R-project.org/</a>.</p>
</div>
<div id="ref-Soe07">
<p>Soebel, Jack D. 2007. “Vulvovaginal Candidosis.” <em>The Lancet</em> 369 (9577): 1961–71. <a href="https://doi.org/10.1016/S0140-6736(07)60917-9">https://doi.org/10.1016/S0140-6736(07)60917-9</a>.</p>
</div>
<div id="ref-Tha06">
<p>Thauer, Rudolf K., and Seigo Shima. 2006. “Methane and Microbes.” <em>Nature</em> 440: 878–79. <a href="https://doi.org/10.1038/440878a">https://doi.org/10.1038/440878a</a>.</p>
</div>
<div id="ref-Wal14">
<p>Waller, Alison S., Takuji Yamada, David M. Kristensen, Jens Roat Kultima, Shinichi Sunagawa, Eugene V. Koonin, and Peer Bork. 2014. “Classification and Quantification of Bacteriophage Taxa in Human Gut Metagenomes.” <em>The ISME Journal</em> 8: 1391–1402. <a href="https://doi.org/10.1038/ismej.2014.30">https://doi.org/10.1038/ismej.2014.30</a>.</p>
</div>
<div id="ref-Wei19">
<p>Wei, Chih-Hsuan, Alexis Allot, Robert Leaman, and Zhiyong Lu. 2019. “PubTator Central: Automated Concept Annotation for.” <em>Nucleic Acids Research</em> 47 (W1): W587–W593. <a href="https://doi.org/10.1093/nar/gkz389">https://doi.org/10.1093/nar/gkz389</a>.</p>
</div>
<div id="ref-Wic18">
<p>Wickham, Hadley, Jim Hester, and Romain Francois. 2018. <em><code>readr</code>: Read Rectangular Text Data</em>. Package version 1.3.1. <a href="https://cran.R-project.org/web/packages/readr/index.html">https://cran.R-project.org/web/packages/readr/index.html</a>.</p>
</div>
<div id="ref-Woe98">
<p>Woese, Carl Richard. 1998. “The Universal Ancestor.” <em>Proceedings of the National Academy of Sciences</em> 95 (12): 6854–9. <a href="https://doi.org/10.1073/pnas.95.12.6854">https://doi.org/10.1073/pnas.95.12.6854</a>.</p>
</div>
<div id="ref-Woe77">
<p>Woese, Carl Richard, and George Edward Fox. 1977. “Phylogenetic Structure of the Prokaryotic Domain: The Primary Kingdoms.” <em>Proceedings of the National Academy of Sciences</em> 74 (11): 5088–90. <a href="https://doi.org/10.1073/pnas.74.11.5088">https://doi.org/10.1073/pnas.74.11.5088</a>.</p>
</div>
<div id="ref-Wyl12">
<p>Wylie, Kristine M., George M. Weinstock, and Gregory A. Storch. 2012. “Emerging View of the Human Virome.” <em>Translational Research</em> 160 (4): 283–90. <a href="https://doi.org/10.1016/j.trsl.2012.03.006">https://doi.org/10.1016/j.trsl.2012.03.006</a>.</p>
</div>
<div id="ref-Xue19">
<p>Xue, Zhuyi. 2019. “<code>ncbitax2lin</code>: Convert NCBI Taxonomy Dump into Lineages.” <a href="https://github.com/zyxue/ncbitax2lin">https://github.com/zyxue/ncbitax2lin</a>.</p>
</div>
</div>
